app/workers/asyncapi/client/job_post_worker.rb

```diff
@@ -8,6 +8,8 @@ def perform(job_id, server_url)
       job = Job.find(job_id)
       server_params = server_params_from(job, job.body)
       response = Typhoeus.post(server_url, {
+        timeout: 60,
```

Comment:
Kind of a similar warning as here: https://github.com/g5search/g5-jobs/pull/183#discussion_r847720962

If this is to set a default then maybe we should use a higher default POST timeout.
@sobrinho did do 60 on the PUT `timeout` though so maybe that's fine https://github.com/G5/asyncapi-server/blob/044c0ecf18ae1276379bcec7fc5c3450e5b9b924/app/workers/asyncapi/server/job_status_notifier_worker.rb#L27-L28

Could probably use a timeout on this delete too https://github.com/G5/asyncapi-client/blob/3bd9a3c9c80ce62704563e7857c844471adb449f/app/workers/asyncapi/client/job_cleaner_worker.rb#L19